conf/metrics.properties.template

```diff
@@ -47,6 +47,10 @@
 #    instance master and applications. MetricsServlet may not be configured by self.
 #
 
+## List of available common sources and their properties.
+
+# org.apache.spark.metrics.source.JvmSource
```

Comment:
This isn't the only `Source`, and this doesn't document anything about it. What's the value?